packages/syft/src/syft/custom_worker/builder.py

```diff
@@ -0,0 +1,98 @@
+# stdlib
+import os.path
+from pathlib import Path
+from typing import Tuple
+
+# third party
+import docker
```

Comment:
Shouldn't `docker` be added to `syft` dependencies?

Reply:
Ouh yes - this is in helm PR, but i missed out to add it here